Sources/LogConsole/LogInterceptor.cs

```diff
@@ -10,6 +10,8 @@
 using System;
 using UnityEngine;
 
+using Logger = KSPDev.LogUtils.Logger;
```

Comment:
Sorry, I didn't know. My aim was just to make the smallest possible changes to avoid introducing any bugs.
How come it wasn't changed already for the various releases for KSP1.1.x then?